pk_tic_tac_toe.rb

```diff
@@ -1,24 +1,36 @@
 class TicTacToeGame
+  VICTORY = [
```

Comment:
Created a constant in the game class so this is accessible throughout the program, and it avoids needing to pass a victory argument into the various methods.